src/main/java/com/mucommander/ui/main/MainFrame.java

```diff
@@ -260,11 +261,16 @@ public void windowClosing(WindowEvent e) {
 
         ActionKeymap.registerActions(this);
 
-        // Fire table change events on registered ActivePanelListener instances, to notify of the intial active table.
+        // Fire table change events on registered ActivePanelListener instances, to notify of the initial active table.
         fireActivePanelChanged(activeTable.getFolderPanel());
 
-        // Set the custom FocusTraversalPolicy that manages focus for both FolderPanel and their subcomponents.
+        // Set the custom FocusTraversalPolicy that manages focus for both FolderPanel and their sub components.
         setFocusTraversalPolicy(new CustomFocusTraversalPolicy());
+
+        // Restore single panel view state
+        if(MuConfigurations.getPreferences().getVariable(MuPreference.USE_SINGLE_PANEL_VIEW, MuPreferences.DEFAULT_USE_SINGLE_PANEL_VIEW)) {
+            ActionManager.performAction(ToggleUseSinglePanelAction.Descriptor.ACTION_ID, this);
+        }
```

Comment:
Yeah, I don't use multiple windows either. I suppose, If I were to open a new window - I am effectively 'cloning' the current set up, right? If you open a new window, you'll also start at your currently open Folders. I would assume that, as a User, I'd also want to start a new windows with my currently selected view settings. 
I.e. when you select to use a single panel view, and start a new frame, you'd want to start with a single panel view as well.
I'm unsure about what would happen if you use the toggle in the second window, you opened, then close it - and then open a new window again. In the first window you'd have your original state, but the settings in this case would already have changed. It gives me a headache.